ui/src/components/content/catalog/components/ServiceDetail.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React from 'react';
+import { Descriptions, Divider, Tag } from 'antd';
+import { CloudUploadOutlined, InfoCircleOutlined } from '@ant-design/icons';
+import { OclDetailVo } from '../../../../xpanse-api/generated';
+import { get, map } from 'lodash';
+
+function ServiceDetail({
+    serviceDetails,
+    serviceRegions,
+}: {
+    serviceDetails: OclDetailVo;
+    serviceRegions: string[];
+}): JSX.Element {
+    return (
+        <>
+            <div className={'catalog-detail-class'}>
+                <h3>
+                    <CloudUploadOutlined />
+                    &nbsp;Available Regions
+                </h3>
+                {map(serviceRegions, (r) => (
```

Comment:
we can use self explaining var names instead of just "r".. applies to everywhere where we have used short var names.

Reply:
fixed.